brainscore_vision/benchmarks/freemanziemba2013/benchmarks/public_benchmarks.py

```diff
@@ -57,34 +48,4 @@ def FreemanZiembaV2PublicBenchmark():
                                stratification_coord='texture_type', bibtex=freemanziemba2013_bibtex)
```

Comment:
shouldn't the FreemanZiemba benchmarks go into their respective plugins as well?